sdk/releases.md

```diff
@@ -0,0 +1,57 @@
+# Ably SDK Team: Guidance on Releases
+
+This document outlines our ideal requirements for SDK releases.
+
+There will be some SDK repositories:
+
+- which are yet to have the work done on them which will implement this
+- where it's necessary to deviate from some of the ideals specified here due to constraints which are innate or otherwise idiomatic to the platform being released for
+
+Where technically possible, SDK repository release processes which claim to adhere to this guidance must follow these ideal requirements closely.
+
+## Release Process
+
+1. Merge all pull requests containing changes intended for this release to `main` branch
+2. Prepare a [Release Branch](#release-branch) and a corresponding pull request, obtain approval from reviewers and then merge to `main` branch
+3. Trigger the [Publish Workflow](#publish-workflow)
+4. If the publish fails then fix the state of the `main` branch, via one or more approved pull requests, with focussed changes that aim to fix the problem and then try step 3 again
+5. Push Git [Version Tag](#version-tag)
+6. Create GitHub release
+
+## Release Branch
+
+- Branch from the `main` branch
+- Merge to the `main` branch, once approved
+- Named like `release/<version>`
+- Increment the version
+- Version should conform to [SemVer](https://semver.org/)
+- Add change log entry (process to be documented under [#17](https://github.com/ably/engineering/issues/17))
+
+## Publish Workflow
+
+- Run in GitHub environment
```

Comment:
There's an implicit "should" or "must" at the beginning of each of these sentences and we should make that explicit. I see you've prefaced the list under 'Version Tag' with 'Should:' so maybe we should do the same here?